modules/installation-osp-creating-network-resources.adoc

```diff
@@ -21,6 +21,12 @@ include::https://raw.githubusercontent.com/openshift/installer/release-4.6/upi/o
 ----
 include::https://raw.githubusercontent.com/openshift/installer/release-4.6/upi/openstack/inventory.yaml[]
 ----
++
+[TIP]
+====
+If you do not define values for`os_api_fip` and `os_ingress_fip`, as well as `os_external_network`, you must perform post-installation network configuration. See "Enabling access to the environment" for more information.
```

Comment:
Refer to last comment about the scenario without os_external_network. bootstrap server will not fetch it's ignition file.